src/main/resources/templates/instructor/course.twig

```diff
@@ -0,0 +1,75 @@
+{% extends 'base_instructor.twig' %}
+
+{% block style %}
+    {{ parent() }}
+    <link href="/css/grid.css" rel="stylesheet"/>
+{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col text-center">
+            <div><h1>{{ course.name }}</h1></div>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col">
+            <div id="grid"></div>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col text-center">
+            <button id="editsave" type="button" class="btn btn-primary">Edit</button>
+        </div>
+    </div>
+{% endblock %}
+
+{% block script %}
+    {{ parent() }}
+    <script type="text/javascript" src="/js/grid.js"></script>
+    <script>
+        $(document).ready(function() {
+            var grid;
+            $.ajax({
+                type: "get",
+                url: "/courses/{{ course.id }}",
+                dataType: "json",
+                success: function(data, status) {
+                    grid = new Grid({
+                        rows: data.rows,
+                        cols: data.cols,
+                        seats: data.seats,
+                        states: ["open", "closed", "reserved"]
+                    });
+                    $('#grid').append(grid.el);
+                },
+                error: function(data, status) {
+                    console.log(data);
+                }
+            });
+
+            $('#editsave').on('click', function(e) {
+                if(grid.editable) {
+                    grid.editable = false;
+                    $('#editsave').html('Edit');
+                    var seats = grid.getSeats();
+                    console.log(seats);
+                    $.ajax({
+                        type: "post",
+                        url: "/courses/{{ course.id }}/seats",
```

Comment:
This ajax call runs when the save button is clicked to save the changes in the databse